src/main/java/com/fabriccommunity/spookytime/client/ClientInitializer.java

```diff
@@ -0,0 +1,10 @@
+package com.fabriccommunity.spookytime.client;
+
+import net.fabricmc.api.ClientModInitializer;
+
+public class ClientInitializer implements ClientModInitializer {
```

Comment:
If you could merge this class with our client initializer we already have, that'd be great!

Reply:
I’ll do that as soon as possible.